pkg/doc/verifiable/test-suite/verifiable_suite_test.go

```diff
@@ -34,35 +34,34 @@ func main() {
 	}
 
 	jwt := flag.String("jwt", "", "base64encoded JSON object containing es256kPrivateKeyJwk and rs256PrivateKeyJwk.")
-	// todo use jwtAud #371
+	// todo use jwtAud #483
 	flag.String("jwt-aud", "", "indication to use aud attribute in all JWTs")
 	jwtNoJws := flag.Bool("jwt-no-jws", false, "indication to suppress the JWS although keys are present")
 	jwtPresentation := flag.Bool("jwt-presentation", false, "indication to generate a verifiable presentation")
 	jwtDecode := flag.Bool("jwt-decode", false, "indication to generate a credential from a JWT verifiable credential. The input file will be a JWT instead of a JSON-LD file.") // nolint: lll
 	isPresentation := flag.Bool("presentation", false, "presentation is passed")
 	flag.Parse()
 
-	if *isPresentation {
-		// todo support Verifiable Presentations #371
-		abort("verifiable presentations are not supported")
-	}
-
 	if *jwt == "" {
-		encodeVCToJSON(vcBytes)
+		if *isPresentation {
+			encodeVPToJSON(vcBytes)
+		} else {
+			encodeVCToJSON(vcBytes)
+		}
 		return
 	}
 
+	if *jwtPresentation {
+		// TODO Encode Verifiable Presentation #483
+		abort("verifiable presentations are not supported")
+	}
+
 	privateKey, publicKey := parseKeys(*jwt)
 
 	if *jwtDecode {
 		decodeVCJWTToJSON(vcBytes, publicKey)
```

Comment:
Are all of those options mutually exclusive? I am asking since only this one doesn't have return.

Reply:
Good catch! The return is missed here.